onnxscript/function_libs/torch_aten/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""A torch function library for onnxscript.
+
+The modules are named after the torch module names for grouping:
+
+- core
```

Comment:
nit: Next time, let's sort them in alphabetical order.

Reply:
Simplified the list